.github/workflows/release_publish.yaml

```diff
@@ -0,0 +1,189 @@
+name: Release Publish
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        required: true
+      increment:
+        type: version
+        description: Bug is for patch and feature for minor
+        options:
+        - bug
+        - feature
+        - major
+
+jobs:
+  release-publish-10.10:
+    name: 'Release Publish Master 10.10'
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master-10.10'
+    env:
+      MAVEN_OPTS: "-Xms1g -Xmx1536m"
+    outputs:
+      version: ${{ steps.project.outputs.version }}
+    steps:
+    - uses: actions/checkout@v3
```

Comment:
don't you need a branch name here. It's a manual workflow if I understand correctly, so it requires more specification

Reply:
its still in test